py3status/modules/file_status.py

```diff
@@ -25,20 +53,23 @@
 {'color': '#FF0000', 'full_text': u'\u25a0'}
 """
 
-from os.path import expanduser, exists
+from glob import glob
+from os.path import basename, expanduser
 
-ERR_NO_PATH = 'no path given'
+STRING_NO_PATH = 'no path given'
```

Comment:
`missing path` for more consistency with other modules.. and feels more friendly.

Reply:
nitpick :)